templates/crate_details.hbs

```diff
@@ -42,8 +42,14 @@
           <li class="pure-menu-item">
             <div class="pure-menu pure-menu-scrollable sub-menu">
               <ul class="pure-menu-list">
-                {{#each versions}}
-                <li class="pure-menu-item"><a href="/crate/{{../name}}/{{this}}" class="pure-menu-link">{{this}}</a></li>
+                {{#each releases}}
+                <li class="pure-menu-item">
+                  {{#if this.build_status}}
+                  <a href="/crate/{{../name}}/{{this.version}}" class="pure-menu-link">{{this.version}}</a>
+                  {{else}}
+                  <a href="/crate/{{../name}}/{{this.version}}" class="pure-menu-link warn"><i class="fa fa-fw fa-warning"></i> {{this.version}} (build failed)</a>
```

Comment:
I don't quite like how this looks. Currently it shows both the warning sign :warning: and `(build failure)` on the right. I would rather have only the warning sign and have the 'build failure' text show up when you hover over it.